src/Microsoft.Health.Fhir.Core/Configs/CoreFeatureConfiguration.cs

```diff
@@ -23,13 +23,23 @@ public class CoreFeatureConfiguration
         public bool SupportsTransaction { get; set; }
 
         /// <summary>
-        /// Gets or sets a value indicating whether to use the ITypedElement based search indexer
+        /// Gets or sets a value indicating whether to use the ITypedElement based search indexer.
         /// </summary>
         public bool UseTypedElementIndexer { get; set; }
 
         /// <summary>
-        /// Gets or sets the default value for IncludeTotal in search bundles
+        /// Gets or sets the default value for IncludeTotal in search bundles.
         /// </summary>
         public TotalType IncludeTotalInBundle { get; set; } = TotalType.None;
+
+        /// <summary>
+        /// Gets or sets the default value for maximum value for _count in search.
+        /// </summary>
+        public int MaxItemCountPerSearch { get; set; } = 100;
+
+        /// <summary>
+        /// Gets or sets the default value for maximum value for _count in search.
+        /// </summary>
+        public int DefaultItemCountPerSearch { get; set; } = 10;
```

Comment:
>DefaultItemCountPerSearch [](start = 19, length = 25)

It wasn't part of issue, but it feels like it also should be configurable.
Also I can't find any validation logic for configuration. Is it suppose to happen during request which relies on configurations? By validation I talk about case where `MaxItemCountPerSearch` is  always not less than `DefaultItemCountPerSearch`